src/xdocs/contributing.xml

```diff
@@ -85,7 +85,7 @@
         <code>mvn -P no-validations install</code>
         <br/>
         Generate uber jar (checkstyle-all-X.XX.jar) to use our
-        <a href="/cmdline.html">command line</a>:
```

Comment:
not related to this issue. The link should be relative as all other links to this page.